### ISSUE TITLE
ModuleNotFoundError in /opt/pav/Snakefile

### ISSUE THREAD

**Issue by GuillaumeHolley** [EichlerLab/pav#62]
Hi,

I am trying to test the latest PAV version from the Singularity image but each time I run it, I get the following error right at the beginning:
```
ModuleNotFoundError in file /opt/pav/Snakefile, line 53:
No module named 'joblib'
  File "/opt/pav/Snakefile", line 53, in <module>
  File "/opt/pav/pavlib/__init__.py", line 7, in <module>
  File "/opt/pav/pavlib/align/__init__.py", line 3, in <module>
  File "/opt/pav/pavlib/align/align.py", line 6, in <module>
  File "/opt/pav/pavlib/seq.py", line 16, in <module>
  File "/opt/pav/dep/svpop/svpoplib/__init__.py", line 24, in <module>
  File "/opt/pav/dep/svpop/svpoplib/svmerge.py", line 7, in <module>
```

The same Singularity command seems to work just fine with an older PAV singularity image I have (2.24.1?).

Thanks,
Guillaume

**Comment by paudano** [comment]
Pull the docker image for version `2.4.1.2` and run it. The joblib module was not installed in previous versions. I think this was an undocumented dependency in one of the libraries PAV uses. PAV doesn't use it directly, and it should have been installed with all of PAV's dependencies, but wasn't. It's now explicitly included in the builds from 2.4.1.2 and later.

**Comment by GuillaumeHolley** [comment]
Hi @paudano,

Thanks for the tip, I specifically pulled the version `2.4.1.2` like you recommended and it worked like a charm.

Thanks again!
Guillaume
